app-java-swing/src/main/java/chessgame/app/swing/controller/BoardController.java

```diff
@@ -58,6 +68,18 @@ public Dimension getSize() {
         return size;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void initMatch() {
+        String[] piece;
+        for (String i : PIECES) {
```

Comment:
For this case you can also use lambda features, eg:

public void initMatch() {
    Stream<String> stream = Arrays.stream(PIECES);
    stream.forEach(p -> {
                String[] piece = p.split(":");
                 view.add(piece[0], piece[1]);
    });
}